### PR TITLE
ROX-19410: Bulk CVE request API

### PR DESCRIPTION
**Pull request by md2119**
## Description

- Added a feature flag for Unified Deferral Workflow.
- Updated the API definition to support bulk CVE operations. With the updated definition, multiple CVEs can be selected in one request by setting `cves` field. The expiry can be set to: 
  - `ALL_CVE_FIXABLE` which indicates the request expires if all CVEs in the request are fixable.
  - `ANY_CVE_FIXABLE` which indicates the request expires if any CVE in the request is fixable. 
For single CVE requests, both have the same effect. Note that this PR only updates the API definition with validation and not the workflow. The workflow will be handled in a separate PR.

## Checklist
- [ ] Investigated and inspected CI test results
- [X] Unit test and regression tests added
- ~Evaluated and added CHANGELOG entry if required~
- ~Determined and documented upgrade steps~
- ~Documented user facing changes~

If any of these don't apply, please comment below.

### Here I tell how I validated my change
Unit tests.

### CODE REVIEW

File: proto/api/v1/vuln_req_service.proto [hunk 1]
```diff
@@ -20,22 +20,46 @@ message ListVulnerabilityRequestsResponse {
     repeated storage.VulnerabilityRequest request_infos = 1;
 }
 
+// next available tag: 8
 message DeferVulnRequest {
-    string                             cve                = 1;
-    string                             comment            = 2;
-    storage.VulnerabilityRequest.Scope scope              = 3;
+    enum ExpiryType {
+        TIME            = 0;
```
Comment (dashrews78) [inline proto/api/v1/vuln_req_service.proto:26]: Should we have an `unset` here for compatibility purposes?  Or is `Time` essentially the default?

File: proto/storage/vuln_requests.proto [hunk 2]
```diff
@@ -31,12 +31,26 @@ enum RequestStatus {
 }
 
 message RequestExpiry {
+    enum ExpiryType {
+        TIME            = 0;
```
Comment (dashrews78) [inline proto/storage/vuln_requests.proto:35]: Should we have an unset here for compatibility purposes? Or is Time essentially the default?

File: proto/api/v1/vuln_req_service.proto [hunk 3]
```diff
@@ -20,22 +20,46 @@ message ListVulnerabilityRequestsResponse {
...
+        ALL_CVE_FIXABLE = 1;
+        ANY_CVE_FIXABLE = 2;
+    }
+
+    // This field indicates the CVEs requested to be deferred.
+    // Only one field, `cve` or `cves`, can be set in the request.
+    string                             cve                      = 1;
+    // This field is under development. This field indicates the CVEs requested to be deferred.
+    // Only one field, `cve` or `cves`, can be set in the request.
+    repeated string                    cves                     = 6;
```
[the remark this answers is not in the document]
Comment (md2119) [reply proto/api/v1/vuln_req_service.proto:36]: `v2` might be overkill. There is no need to deprecate the `cve`. We can continue to support both fields as long as the API docs inform users how to use the field as on `ln35`.

**Review by dashrews78 — APPROVED** [review]
Looks good assuming the UT failures don't drive significant change.  I assume they won't.

**Comment by md2119** [comment]
@dashrews78 I undid the following proto and other related changes because it is not required. The original data model that I added back then satisfies and works fine.
```
message global {
  bool enabled = 1;
}

**Pull request merged**